MVC/controllers/topicsController.js

```diff
@@ -1,9 +1,32 @@
-const {selectAllTopics} = require('../models/topicsModel');
+const { selectAllTopics } = require("../models/topicsModel");
 
 exports.getAllTopics = (req, res, next) => {
   selectAllTopics()
-      .then((topics) => {
-        res.status(200).send({topics});
-      })
-      .catch(next);
-  }
\ No newline at end of file
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch(next);
+};
+
+exports.getAllEndpoints = (req, res, next) => {
+  res.status(200).send({
+    endpoints: {
+      "GET /api/topics": {
```

Comment:
This is fine for now but moving forward you will find you have to go and update this every time. Remember, we already have this data somewhere. What could we send instead?